src/platform/qt/AssetView.cpp

```diff
@@ -21,7 +21,7 @@ AssetView::AssetView(GameController* controller, QWidget* parent)
 	connect(&m_updateTimer, SIGNAL(timeout()), this, SLOT(updateTiles()));
 
 	connect(m_controller, &GameController::frameAvailable, &m_updateTimer,
-	        static_cast<void(QTimer::*)()>(&QTimer::start));
+	        static_cast<void(QTimer::*)()>(&QTimer::start), Qt::QueuedConnection);
```

Comment:
I highly doubt this will matter. Since it's a cross-thread call the default is already QueuedConnection.